src/Dock.Avalonia/DockControlState.cs

```diff
@@ -56,7 +56,7 @@ private void Drop(Point point, DragAction dragAction, IVisual relativeTo)
                 operation = target.GetDockOperation(point, relativeTo, dragAction, Validate);
             }
 
-            if (_dropControl is DockPanel)
+            if (_dropControl is Panel)
```

Comment:
We probably need to add custom Panel type here or something otherwise dock target is displayed on tabitem header